modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -49,11 +48,10 @@ func NewKeeperWithVM(
 	}
 
 	ibcwasm.SetVM(vm)
-	ibcwasm.SetWasmStoreKey(key)
+	ibcwasm.SetWasmStoreService(storeService)
```

Comment:
should we do a check here to make sure that the store service isn't `nil`?

Reply:
yea, I was also thinking if we can just pass store key here and not give away what we use as storage in our public API.

Reply:
Many SDK modules use storeService not storeKeys anymore.

Reply:
added the nil check and test case